### PR TITLE
DISPATCH-1680 add openproperties doc

### PR DESCRIPTION
**Pull request by pwright**
@kgiusti could you take a look please

### CODE REVIEW

File: docs/books/modules/user-guide/adding-metadata-to-connections.adoc [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+[options="nowrap",subs="+quotes"]
+----
+connector {
+   name: broker
+   role: route-container
+   host: 127.0.0.1
+   port: 22180
+   saslMechanisms: ANONYMOUS
+   openProperties: {
+      "label": "green"
```
Comment (jiridanek) [inline docs/books/modules/user-guide/adding-metadata-to-connections.adoc:47]: missing brace
```suggestion
      "label": "green"
  }
```

Comment (jiridanek) [inline docs/books/modules/user-guide/adding-metadata-to-connections.adoc:47]: missing `}`
```suggestion
   openProperties: {
      "label": "green"
   }
```

Comment (pwright) [reply docs/books/modules/user-guide/adding-metadata-to-connections.adoc:47]: ah, thanks

File: docs/books/modules/user-guide/adding-metadata-to-connections.adoc [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+   }
+}
+----
+
+Note the following restrictions on the JSON entries:
+ 
+* ASCII characters only for keys
+* The following keys are not allowed:
+** qd.*
```
Comment (kgiusti) [inline docs/books/modules/user-guide/adding-metadata-to-connections.adoc:56]: On re-reading this I'm concerned a reader may take the asterisk literally, like "avoid 'qd.*' but 'qd.foobar' is just fine."

Should probably state explicitly that keys starting with the prefix "qd." and "x-opt-qd." are reserved and should not be used.

**Pull request merged**